package.json

```diff
@@ -41,19 +41,22 @@
     },
     "lint-staged": {
         "src/**/*.{ts,tsx}": [
-          "eslint 'src/**/*.tsx' --fix", "git add"
+            "eslint 'src/**/*.tsx' --fix",
+            "git add"
         ],
         "src/**/*.scss": [
-          "stylelint 'src/**/*.tsx' --fix", "git add"
+            "stylelint 'src/**/*.tsx' --fix",
+            "git add"
         ]
     },
     "dependencies": {
         "@medly-components/core": "^1.13.0",
         "@medly-components/forms": "^1.5.1",
         "@medly-components/layout": "^1.6.0",
-        "@medly-components/theme": "^1.8.0",
         "@medly-components/loaders": "^1.0.12",
+        "@medly-components/theme": "^1.8.0",
         "axios": "^0.19.2",
+        "dotenv-webpack": "^2.0.0",
```

Comment:
It should not be present in dependencies, its only required during build.

So, its just a devDependency